src/Xlang/README.md

```diff
@@ -0,0 +1,19 @@
+## Microsoft/Xlang/UndockedRegFreeWinRT

```

Comment:
>## [](http://example.com/codeflow?start=0&length=2)

You should also say that if there's an update from the subtree the PR shouldn't be squashed

Reply:
Added suggested comment.